Examples/Python/cinematic_rendering/thorax_kidneys.py

```diff
@@ -0,0 +1,139 @@
+from pathlib import Path
+import signal
+import sys
+
+import vtk
+
+from vtk import vtkRenderingLookingGlass
+
+# Kill the program when ctrl-c is used
+signal.signal(signal.SIGINT, signal.SIG_DFL)
+
+data_url = "https://data.kitware.com/api/v1/file/62d9672ebddec9d0c4af738a/download"
+data_file = "thorax_kidneys.vti"
+
+if not Path(data_file).exists():
+    print(f"Downloading data to {data_file}")
+
+    # Download the data
+    import urllib.request
+    urllib.request.urlretrieve(data_url, data_file)
+
+    print("Download complete")
+
+# Volumetric shadow settings
+volumetric_scattering_blending = 2
+global_illumination_reach = 0.35964
+
+# Read the data
+reader = vtk.vtkXMLImageDataReader()
+reader.SetFileName("thorax_kidneys.vti")
```

Comment:
Should  use data_file var as arg to SetFileName

Reply:
Good point!

Reply:
Fixed